metazoa/species/Bemisia/Bemisia_tabaci/Bemisia_tabaci_gca903994095_assembly.md

```diff
@@ -0,0 +1,9 @@
+**Assembly**
+------------------------
+The *Bemisia tabaci* Uganda 1 genome was produced by the genomics consortium of the [African Cassava Whitefly Project](http://www.cassavawhitefly.org), funded by the Bill & Melinda Gates Foundation (Grant Number OPP1058938). 
+
+A field-collected colony (Namulonge, Uganda) was established, maintained and inbred \(F6 generation\) by [Dr Habibu Mugerwa](https://scholar.google.com/citations?user=srDeBUgAAAAJ&hl=en) at the quarantine insectary facilities of the [Natural Resources Institute, University of Greenwich, United Kingdom](https://www.nri.org/). 
```

Comment:
Could you please delete the backslashes before the parentheses (`\(F6 generation\)`)?